doc/tutorials/cne/cne.txt

```diff
@@ -0,0 +1,356 @@
+/*!
+
+@file cne.txt
+@author Kartik Nighania
+@brief Tutorial on how to use the CNE optimizer class.
+
+@page cnetutorial CNE Optimizer tutorial (mlpack_cne)
+
+@section intro_cnetut Introduction
+
+Conventional Neural Evolution (CNE) is a class of evolutionary algorithms
+focused on dealing with fixed topology networks. This class implements this
```

Comment:
Which class is `this class`?  I'm not sure here, maybe you can reference the class the tutorial is about?